src/libraries/System.Runtime/tests/TrimmingTests/DefaultValueAttributeCtorTest.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
```

Comment:
This test maps to https://github.com/dotnet/runtime/pull/37402/files#diff-7cf757bfbf7fb9fd9b21ef5383d89b30R32.